File: backend/rentify-backend/src/main/java/com/rentify/model/BusinessResponse.java
package com.rentify.model;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.Setter;


@Getter
@Setter
@AllArgsConstructor
@Builder
public class BusinessResponse {
    private Long id;
    private String businessName;
    private String businessType;
    private String description;
    private String contactEmail;
    private String phone;
    private String storeSlug;
    private StoreThemeResponse storeTheme;
    private boolean isProductAvailable;
}

File: frontend/rentify-frontend/src/app/models/business-response.ts
import { StoreThemeResponse } from "./store-theme-response";

export interface BusinessResponse {
  id: number;
  businessName: string;
  businessType: string;
  description: string;
  contactEmail: string;
  phone: string;
  storeSlug: string;
  storeTheme: StoreThemeResponse;  
  isProductAvailable: boolean;
}
